dts/bindings/sensor/st,lis2dux12-common.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2023 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+include: sensor-device.yaml
+
+properties:
+  irq-gpios:
```

Comment:
I think we can have here int1-gpios and int2-gpios, like other ST sensors already in main. My opinion is that is better, since it is more clear in the association. It is true that irq-gpios is an array, but I'm not sure how to assign int1 and int2 in the DT when only one is connected (let's say INT2).

Reply:
Alright. I split these into `int1-gpios`, and `int2-gpios`, as you suggested.